python/lsst/afw/image/testUtils.py

```diff
@@ -23,8 +23,11 @@
 
 # the asserts are automatically imported so unit tests can find them without special imports;
 # the other functions are hidden unless explicitly asked for
-__all__ = ["assertImagesNearlyEqual", "assertMasksEqual", "assertMaskedImagesNearlyEqual",
-           "assertImagesEqual", "assertMaskedImagesEqual"]
+__all__ = ["assertImagesAlmostEqual", "assertImagesEqual", "assertMasksEqual",
```

Comment:
when modifying the all line, you might as well move it to the proper place after imports